### ISSUE TITLE
Erroneous warning given about block imports by ImportsChecker 

### ISSUE THREAD

**Issue by espertus** [scalastyle/scalastyle#316]
For this code:
```
import a.A
import a.{B, C}
```
I get this warning:
```
<error column="0" line="2" source="org.scalastyle.scalariform.ImportOrderChecker" severity="warning" message="a. is in wrong order relative to a.A."/>
```

I used this configuration:
```
 <check class="org.scalastyle.scalariform.ImportOrderChecker"  level="warning" enabled="true">
  <parameters>
   <parameter name="lexicographic">true</parameter>
  </parameters>
 </check>
```

We do not use `BlockImportChecker` since my team allows block imports.

No errors are reported if I switch the order of the imports:
```
import a.{B, C}
import a.A
```

The behavior I expected was for the block import to be expanded internally, yielding no warnings for the original version and a warning for the switched version.
